config/config.go

```diff
@@ -57,6 +57,7 @@ type ServerConfig struct {
 	DbSslCertContents     string `ini:"db_sslcert_contents"`
 	DbSslKey              string `ini:"db_sslkey"`
 	DbSslKeyContents      string `ini:"db_sslkey_contents"`
+	DbUseIamAuth          bool   `ini:"db_use_iam_auth"`
```

Comment:
Note for reviewer: This is intentionally not in the AWS config variable section (and not prefixed with `aws_`), since Google Cloud SQL and Azure Database both support IAM authentication as well, and we can control whether that is used with the same config variable in the future.